Wikipedia/Code/WMFNewArticleTabsSettingsViewController.swift

```diff
@@ -0,0 +1,103 @@
+import UIKit
+import SwiftUI
+import WMFComponents
+import WMFData
+
+class WMFNewArticleTabsSettingsViewController: UIViewController {
+    private var hostingController: UIHostingController<WMFNewArticleTabSettingsView>?
+    private var viewModel: WMFNewArticleTabSettingsViewModel?
+    private let userDefaultsStore = WMFDataEnvironment.current.userDefaultsStore
+    let dataController = WMFArticleTabsDataController()
+    private var dataStore: MWKDataStore?
```

Comment:
You might be able to remove this legacy data store dependency. You're getting all of your data from our newer areas (WMFData), which is good!